src/__forks__/Promise.native.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *
+ * Copyright 2013-2014 Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This was an oversight on my part. I copied and pasted from RN, and for some reason this had Apache license. Will fix.